jsonnet/dashboards/apps/ssl-exporter.libsonnet

```
@@ -0,0 +1,253 @@
+/*
+  Copyright 2020 The dNation Kubernetes Monitoring Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local prometheus = grafana.prometheus;
+local row = grafana.row;
+local table = grafana.tablePanel;
+local statPanel = grafana.statPanel;
+
+{
+  grafanaDashboards+:: {
+
+    'ssl-exporter':
+      local totalUniqueCerts = statPanel.new(
```

To be uniform with other dashboards, please change it like I suggest. Change it also for other panels.
```suggestion
      local totalUniqueCerts =
        statPanel.new(
```